examples/swift/greeter.swift

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env xcrun -sdk macosx swift
+
+import Foundation
+
+while(true){
```

Comment:
Curious... how does this work? If no messages are incoming, won't it just spin repeatedly printing "Hello !"? Does something block until no data is available?